canonical-websites/netplan.io
# Rename sections, update reference doc

## Done

- renamed sections to something that might be closer to end result, but at least more sensical than their current state (as per Budapest meeting)
- update reference doc; move it to markdown

This means updating the reference from now on should be as simple as copying the markdown document from gh:CanonicalLtd/netplan.

## QA

Open the website, check Reference and Examples section exists and look reasonable.
